react/features/toolbox/components/web/Toolbox.js

```diff
@@ -1055,9 +1000,9 @@ class Toolbox extends Component<Props> {
             (_liveStreamingEnabled || _liveStreamingDisabledTooltipKey)
                 && this._shouldShowButton('livestreaming')
                 && this._renderLiveStreamingButton(),
-            (_fileRecordingsEnabled || _fileRecordingsDisabledTooltipKey)
-                && this._shouldShowButton('recording')
```

Comment:
I think this check for making sure the button is in interfaceConfig TOOLBAR_BUTTONS might have gotten removed. Web has interfaceConfig.TOOLBAR_BUTTONS to let people declare which buttons are visible and previously the order of the buttons. (It's weird having button displayed configured based only on fileRecordingsEnabled when other buttons rely on interfaceConfig, although it's also weird that it needs to be declared as enabled in both config and interfaceConfig...)

Reply:
We should use `this._shouldShowButton('recording')` as the `visible` prop to the button.

Reply:
Right, thanks for pointing it out, I left that check, I'll add now.

On a semi-related note: why do we need interfaceConfig if we don't do such thing on mobile? If interfaceConfig affects behaviour on web but is not available on mobile, we'll surely end up with different behaviour there.

Reply:
Because the web allows for hiding buttons and mobile doesn't. Arguably it's mobile the one who is behind here, because we don't load `interface_config.js`. Yet.